src/config/colors.toml

```diff
@@ -19,6 +19,7 @@
 "divergent" = "red"
 "divergent change_id"="red"
 "conflict" = "red"
+"empty" = "yellow"
```

Comment:
Maybe we should add a `"working_copy empty" = "bright yellow"` for consistency, so everything on the working-copy commit lines is bright? That also reminds me of #528...

Reply:
Sure!